src/varlink.js

```diff
@@ -83,6 +91,17 @@ function connect(address) {
         return new Promise((resolve, reject) => pending.push({ resolve, reject }));
     };
 
+    connection.monitor = function (method, parameters, eventCallback) {
```

Comment:
Varlink calls everything "reply", not "event". Please stay with that terminology and use `eventCallback` instead of `replyCallback`.

Reply:
Fixed. (I am sure you meant to use replyCallback btw instead of eventCallback)